plugins/woo-ai/src/image-background-removal/background-removal-link.tsx

```diff
@@ -134,6 +150,42 @@ export const BackgroundRemovalLink = () => {
 				</button>
 				<img src={ MagicIcon } alt="" />
 			</div>
+			{ ! hasBeenDismissedBefore && (
+				<TourSpotlight
+					id="backgroundRemovalLink"
+					reference={ `#${ LINK_CONTAINER_ID }` }
+					description={ __(
+						'Effortlessly make your product images pop by removing the background using state-of-the-art AI technology. Just click the button and watch!',
+						'woocommerce'
+					) }
+					title={ createInterpolateElement(
+						__(
+							'<NewBlock /> Remove backgrounds with AI',
+							'woocommerce'
+						),
+						{
+							NewBlock: (
+								<span className="woo-ai-background-removal-link__new-block">
+									{ __( 'NEW', 'woocommerce' ) }
+								</span>
+							),
+						}
+					) }
+					placement="left"
+					spotlightParent={
```

Comment:
There is some oddness that crops up with React root's nested within Backbon JS templates, and putting this within the same container tends to resolve it.